tests/test_files/cicsGenApp/positive/lgtestp1.cbl

```diff
@@ -45,6 +45,7 @@
 
       * Display Main Menu
            EXEC CICS SEND MAP ('SSMAPP1')
+                     MAPONLY
```

Comment:
The call without MAPONLY seems valid. Please revert changes in these test files.